client/my-sites/customer-home/cards/tasks/task.jsx

```diff
@@ -90,6 +90,14 @@ const Task = ( {
 		);
 	};
 
+	const ActionButtonWithStats = ( { children } ) => {
+		return (
+			<div onClick={ startTask } role="presentation">
```

Comment:
I think this works for now @kwight! We might be able to swap with a React Fragment later if they add support for event handlers.

https://reactjs.org/docs/fragments.html